parts/chapters/subsections/10.3/SKIP100.fodt

```diff
@@ -4001,8 +4001,8 @@ Updated with AFR/TSA Rev-D comments and new keywords.</dc:description><meta:init
    <text:section text:style-name="Sect1" text:name="SKIP100">
 
 <text:h text:style-name="P17643" text:outline-level="3"><text:bookmark-start text:name="__RefHeading___Toc65046_1640804870141"/><text:span text:style-name="T1">SKIP</text:span>100<text:span text:style-name="T1"> – Activate Skipping of </text:span>Black-Oil Keywords and Input Data<text:bookmark-end text:name="__RefHeading___Toc65046_1640804870141"/></text:h>
-    <text:p text:style-name="P12355">This keyword turns on skipping of black-oil keywords until the ENDSKIP activation keyword is encountered. All black-oil keywords between the SKIP100 and ENDSKIP keywords are ignored by OPM Flow. </text:p>
-    <text:p text:style-name="P12349">See <text:a xlink:type="simple" xlink:href="#3.2.32.SKIP100 – Activate Skipping of “Black-Oil” Keywords and Input Date|outline" text:style-name="Internet_20_link" text:visited-style-name="Visited_20_Internet_20_Link"><text:bookmark-ref text:reference-format="text" text:ref-name="__RefHeading___Toc65046_1640804870">SKIP100 – Activate Skipping of Black-Oil Keywords and Input Data</text:bookmark-ref></text:a><text:s/>in the GLOBAL section for a full description.</text:p>
+    <text:p text:style-name="P12355">The SKIP100 keyword activates skipping of all keywords and input data by the commercial black-oil simulator until the ENDSKIP keyword is encountered. All keywords between the SKIP100 and ENDSKIP keywords are ignored by the commercial black-oil simulator. The SKIP100 keyword is ignored by the commercial compositional simulator. </text:p>
```

Comment:
Why are we talkig about "commercial simulator" here and below?

Reply:
ping. Is this intentional?

Reply:
For more details see [here](https://github.com/OPM/opm-simulators/pull/5085#issuecomment-2066753541)